db/seeds.rb

```diff
@@ -109,6 +109,7 @@
   r.url = "http://www.dot.ri.gov/img/travel/camimages/95-21%20I-95%20S%20@%20Broad%20St%20.jpg"
   r.location = providence
   r.feed_format = "image"
+  r.source_url = "http://www.dot.ri.gov"
```

Comment:
I don't think we need an extra column for this, if it's just the base url for the url we are already keeping. You can parse this from the js side like this:

```js
const url = "http://www.dot.ri.gov/img/travel/camimages/95-21%20I-95%20S%20@%20Broad%20St%20.jpg"
const source_url = new URL(url).origin
```